spec/queries/languages/get_languages_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+module Queries
+  RSpec.describe Types::QueryType, type: :request do
+    before :each do
+      @language_one = create(:language)
+      @language_two = create(:language)
+      @language_three = create(:language)
+    end
+
+    def query
+      <<~GQL
+        query {
+          getLanguages{
+            id
+            name
+          }
+        }
+      GQL
+    end
+
+    it 'can get all languages' do
+      post '/graphql', params: { query: query }
+
+      json = JSON.parse(response.body)
```

Comment:
```suggestion
      json = JSON.parse(response.body, symbolize_names: true)
```

Reply:
Want to start symbolizing the response hashes? You'd have to change the lines below to : instead of ' but i think it can look cleaner.

Reply:
Yeah I always forget that's a thing